draft-pardue-moq-qlog-moq-events.md

```diff
@@ -285,7 +285,10 @@ MOQTObjectDatagramParsed = {
 
 The `subgroup_header_created` event is emitted when a stream begins and a
 SUBGROUP_HEADER is created. It has Core importance level; see {{Section 9.2 of
-QLOG-MAIN}}.
+QLOG-MAIN}}. The SUBGROUP_HEADER object in MoqT uses 12 type values to encode
+various properites. The event conveys these as explicit fields, such as
+contains_end_of_group. If the subgroup_id is the object_id of the first object,
+the subgroup_id is omitted. Otherwise, it is included with the relevant value.
```

Comment:
```suggestion
QLOG-MAIN}}. 

The SUBGROUP_HEADER object in MoQT uses 12 type values to encode various
properties. The `subgroup_header_created` event conveys these as explicit
fields, such as contains_end_of_group.

If the subgroup_id is the object_id of the first object, the subgroup_id is
omitted. Otherwise, it is included with the relevant value.
```